### PR TITLE
[fx] added activation checkpointing annotation

### PR DESCRIPTION
**Pull request by FrankLeeeee**

### CODE REVIEW

File: tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+def test_activation_checkpoint_annotation():
+    module = MyModule()
+
+    # test tracing with activation checkpoint
+    tracer = ColoTracer(trace_act_ckpt=True)
+    graph = tracer.trace(module)
+    gm = GraphModule(module, graph)
+
+    # print([f"{node.name} cktp = {getattr(node, 'ckpt', False)}" for node in gm.graph.nodes])
```
Comment (YuliangLiu0306) [inline tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py:45]: Delete this line.

File: tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import torch
+import torch.nn as nn
+from colossalai.fx import ColoTracer
+from torch.fx import GraphModule
+from torch.utils.checkpoint import checkpoint
+
+
+class MLP(torch.nn.Module):
```
Comment (YuliangLiu0306) [inline tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py:8]: This model is not used in this test.

File: tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py (hunk 1 above)
Comment (FrankLeeeee) [reply tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py]: done.

File: tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py (hunk 2 above)
Comment (YuliangLiu0306) [reply tests/test_fx/test_tracer/test_activation_checkpoint_annotation.py:8]: okay

**Review by YuliangLiu0306 — APPROVED** [review]

**Pull request merged**